musescore/MuseScore
# Allow reversal of hook flags within Properties: Chord

Port of https://github.com/Jojo-Schmitz/MuseScore/pull/742

Resolves: https://musescore.org/en/node/109706

Provides chord inspector checkbox to reverse hooks.

To test:

1. Select a note head
2. Make sure it's of the flag duration type 1/8th or lesser in value
3. Enable "reversed hook"

![image](https://github.com/user-attachments/assets/8b76844a-83c4-49d7-9d13-57daeab33d6c)
(Image from MuseScore 3.7)

Kind of a gimmick, but so long as it doesn't hurt anything I suppose it might fulfill little niche use cases for some people.

Testing appreciated
